atc/atccmd/command.go

```diff
@@ -259,6 +259,10 @@ type RunCommand struct {
 	P2pVolumeStreamingTimeout time.Duration `long:"p2p-volume-streaming-timeout" description:"Timeout value of p2p volume streaming" default:"15m"`
 
 	DisplayUserIdPerConnector map[string]string `long:"display-user-id-per-connector" description:"Define how to display user ID for each authentication connector. Format is <connector>:<fieldname>. Valid field names are user_id, name, username and email, where name maps to claims field username, and username maps to claims field preferred username"`
+
+	DefaultGetTimeout  time.Duration `long:"default-get-timeout" description:"Default timeout of get steps" default:"1h"`
+	DefaultPutTimeout  time.Duration `long:"default-put-timeout" description:"Default timeout of put steps" default:"1h"`
+	DefaultTaskTimeout time.Duration `long:"default-task-timeout" description:"Default timeout of task steps" default:"10h"`
```

Comment:
@taylorsilva For sure, I can remove the default values. And yes, some pre-communications are needed. Actually, we are doing such communications before the tool is place. But I believe in any case a big enough threshold is necessary. Because pipelines run in background, not all pipeline owners closely monitor their pipelines, especially on large deployments. Not any administrator wants a `task` to run 1 week long. And Concourse doesn't provide a tool to help find out such stuck builds, what I was done was to run SQL statements over the db directly, which requires certain understanding on Concourse db structure. For those Concourse administrators who don't have the ability, they may even not be able to identify stuck builds. If there are max timeout settings, pipeline may fail, then pipeline owners can set `Timeout` on certain steps to overwrite default timeout.